modules/hcp-enable-feature-sets.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-using-feature-gates.adoc
+
+:_mod-docs-content-type: PREFERENCE
+[id="hcp-enable-feature-sets_{context}"]
+= Enabling feature sets using feature gates
+
+You can use the OpenShift CLI (`oc`) to enable the `TechPreviewNoUpgrade` feature set in a hosted cluster by editing the `HostedCluster` custom resource (CR).
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Open the `HostedCluster` CR for editing by running the following command:
++
+[source,terminal]
+----
+$ oc edit <hosted_cluster_name> -n clusters <1>
+----
+<1> Replace `<hosted_cluster_name>` with your hosted cluster name.
++
+.Sample `HostedCluster` CR
+[source,yaml]
+----
+apiVersion: hypershift.openshift.io/v1beta1
+kind: HostedCluster
+metadata:
+  name: <hosted_cluster_name> <1>
+  namespace: clusters
+spec:
+  configuration:
+    featureSet: TechPreviewNoUpgrade <2>
+----
+<1> Specifies your hosted cluster name.
+<2> This feature set is a subset of the current Technology Preview features. This feature set allows you to enable these Technology Preview features on test clusters, where you can fully test them, while leaving the features disabled on production clusters.
++
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. You should not enable this feature set on production clusters.
+====
+
+. Save the file to apply the changes.
+
+.Verification
+
+* Verify that the `TechPreviewNoUpgrade` feature gate is enabled by running the following command:
++
+[source,terminal]
+----
+$ oc get featuregate <hosted_cluster_name> -o yaml <1>
```

Comment:
It isn't clear that the `oc edit` command must happen on the _hosting_ cluster and this `oc get` command needs to be run from within the _hosted_ cluster.  These are two different clusters.